elpis/engines/common/objects/command.py

```diff
@@ -11,9 +11,9 @@ def run(cmdline: str) -> subprocess.CompletedProcess:
     args = ['bash', '-c', cmdline]
     process = subprocess.run(
         args,
-        check=True,
+        check=False,
```

Comment:
Do you want it to fail and continue running, rather than throwing an exception?

Reply:
I changed back to check True, and wrapped the train and transcribe run commands in try:except things.